core/example/solvers/ewald.cpp

```diff
@@ -0,0 +1,270 @@
+#include "definitions.h"
+#include "ewald.h"
+#include <cmath>
+#include <gsl/gsl_sf_lambert.h>
```

Comment:
You need to find the GSL package at configuration time and register the dependency properly if you want to use it.
Also you don't seem to be using it at the moment.  I realize now this is a WIP.  Will hold off reviewing unless you want early feedback.